components/brave_shields/browser/ad_block_regional_service_manager.cc

```diff
@@ -205,6 +205,35 @@ void AdBlockRegionalServiceManager::AddResources(const std::string& resources) {
   }
 }
 
+bool AdBlockRegionalServiceManager::IsFilterListAvailable(
+    const std::string& uuid) const {
+  DCHECK(!uuid.empty());
+  auto catalog_entry =
+      brave_shields::FindAdBlockFilterListByUUID(regional_catalog_, uuid);
+  return catalog_entry != regional_catalog_.end();
+}
+
+bool AdBlockRegionalServiceManager::IsFilterListEnabled(
+    const std::string& uuid) const {
+  DCHECK(!uuid.empty());
+  DCHECK(local_state_);
+
+  if (uuid == kCookieListUuid &&
+      base::FeatureList::IsEnabled(kBraveAdblockCookieListDefault) &&
+      !local_state_->GetBoolean(prefs::kAdBlockCookieListSettingTouched)) {
+    return true;
+  }
+
+  const auto& regional_filters_dict =
+      local_state_->GetDictionary(prefs::kAdBlockRegionalFilters)->GetDict();
+
+  if (const auto* regional_filter_dict = regional_filters_dict.FindDict(uuid)) {
+    return regional_filter_dict->FindBool("enabled").value_or(false);
```

Comment:
this is not the third call to `regional_filter_dict->FindBool("enabled").value_or(false);` so maybe we should create some helper methods? Or maybe these are the helper methods and they should be used internally in place of existing code?

Reply:
oh, I guess I didn't look down far enough :)